xc/xc7/tests/soc/litex/base/README.md

```diff
@@ -23,3 +23,7 @@ export CPU_VARIANT=linux
 export PLATFORM=arty
 export FIRMWARE=linux
 ```
+
+To create `baselitex_arty100t.v`, copy from `baselitex_arty.v` and then make the following edits:
+* *Add* constraint `(* LOC="PLLE2_ADV_X1Y1" *)` to instance `PLLE2_ADV`.
```

Comment:
This shouldn't be required, we should identify and fix the underlying bug that resulted in this workaround.

Reply:
I agree, I filed issue #1542 last week on this issue.   When this test is merged, I'll mention it in the issue so that we remember to come back and take out the workaround when the issue is fixed.